### PR TITLE
Chore: Use enter_exit_debug_log decorator

### PR DESCRIPTION
**Pull request by OjusWiZard**
## Checklist

- [x] Use `enter_exit_debug_log` decorator in data migrations, UserDB, and GlobalDB upgrades.

### CODE REVIEW

File: rotkehlchen/data_migrations/migrations/migration_10.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
 log = RotkehlchenLogsAdapter(logger)
 
 
+@enter_exit_debug_log()
```
Comment (yabirgb) [inline rotkehlchen/data_migrations/migrations/migration_10.py:17]: can't you remove log from here?

File: rotkehlchen/data_migrations/migrations/migration_11.py [hunk 2]
```diff
@@ -18,12 +18,11 @@
 log = RotkehlchenLogsAdapter(logger)
 
 
+@enter_exit_debug_log()
 def data_migration_11(rotki: 'Rotkehlchen', progress_handler: 'MigrationProgressHandler') -> None:
```
Comment (yabirgb) [inline rotkehlchen/data_migrations/migrations/migration_11.py:22]: same remove the log = Rot..

File: rotkehlchen/data_migrations/migrations/migrations_13.py [hunk 3]
```diff
@@ -18,13 +18,13 @@
...
 
 
+@enter_exit_debug_log()
 def data_migration_13(rotki: 'Rotkehlchen', progress_handler: 'MigrationProgressHandler') -> None:
     """
     Introduced at v1.31.0
 
     Detects Gnosis and Base chain accounts that have activity and are not yet tracked.
     """
-    log.debug('Enter data_migration_13')
```
Comment (yabirgb) [inline rotkehlchen/data_migrations/migrations/migrations_13.py:27]: remove the log object

File: rotkehlchen/db/upgrades/v37_v38.py [hunk 4]
```diff
@@ -30,10 +30,12 @@
 ]
 
 
+@enter_exit_debug_log()
 def _add_polygon_pos_location(write_cursor: 'DBCursor') -> None:
```
Comment (yabirgb) [inline rotkehlchen/db/upgrades/v37_v38.py:34]: you can remove the log object of this file

File: rotkehlchen/db/upgrades/v36_v37.py [hunk 5]
```diff
@@ -232,7 +224,7 @@ def _fix_kraken_events(write_cursor: 'DBCursor') -> None:
     - instant swaps
     Needs to be executed after _update_history_events_schema
     """
-    log.debug('Enter _fix_kraken_events. Fixing kraken eth2 related tuples')
+    log.debug('Fixing kraken eth2 related tuples')
```
Comment (LefterisJP) [inline rotkehlchen/db/upgrades/v36_v37.py:227]: remove this

File: rotkehlchen/globaldb/upgrades/v2_v3.py [hunk 6]
```diff
@@ -370,9 +371,9 @@ def translate_assets_in_price_table(cursor: 'DBCursor') -> list[tuple[str, str,
     return updated_rows
 
 
+@enter_exit_debug_log(name='GlobalDB v2->v3 upgrade')
 def migrate_to_v3(connection: 'DBConnection') -> None:
     """Upgrade assets information and migrate globaldb to version 3"""
-    log.debug('Entered globaldb v2->v3 upgrade')
```
Comment (LefterisJP) [inline rotkehlchen/globaldb/upgrades/v2_v3.py:375]: you also need to remove `log.debug('Finished globaldb v2->v3 upgrade')` from line 550

**Review by LefterisJP — APPROVED** [review]
LGTM

**Pull request merged**